azavea/raster-foundry
# Add json definition to tool model

## Overview

This PR adds a column to tools called `definition` of type `JSONB`. It also includes a migration to add the current NDVI difference tool definition.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [] Swagger specification updated, if necessary~

### Demo

<img width="718" alt="screen shot 2017-02-10 at 7 00 24 pm" src="https://cloud.githubusercontent.com/assets/2442245/22848792/f8b3bbf0-efc4-11e6-9638-c029d4495d52.png">

## Testing Instructions

 * Run migrations (`./scripts/console app-server ./sbt`, `mg update`, `mg apply`)
 * Test (`project/app`, `test`)
 * Run (exit and `./scripts/server`)
 * Go to the lab and select the NDVI difference tool
 * Check your requests in dev tools and notice that the tool definition was brought along (as shown above)
